.github/workflows/build.yml

```diff
@@ -29,4 +29,4 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-gradle-
       - run: |
-          ./gradlew build --no-daemon
+          ./gradlew build --no-daemon -DVERSION=0.0.0
```

Comment:
I see no point doing this. Is there one ?

Reply:
fixed with proposed fix above